### PR TITLE
LP-79: Entity Share for transferring content from Preprod to Prod

### PR DESCRIPTION
**Pull request by Polynya**
## Include a summary of what this merge request involves (*)
Fostering content has been created on Preprod and must be transferred to Prod.

This PR installs the modules required, including Entity Share, JsonAPI Extras, Simple OAuth. Only JsonAPI Extras and its dependencies have been enabled to mimimise the configuration - Preprod and Prod will be set up differently (server and client) and the modules will be uninstalled afterwards.

## Call out any relevant implementation decisions
- Are there any links to back up your chosen methodology?
- Why have you taken the approach?
- Any particular problem areas?
## This PR has been tested in the following browsers
- [ ] Arc
- [ ] Edge
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] IE 11 (Windows)
- [ ] iOS Chrome
- [ ] iOS Safari
- [ ] Android Chrome
- [ ] Android Firefox
- [ ] Android default

### CODE REVIEW

File: config/default/core.extension.yml
Comment (siliconmeadow) [inline config/default/core.extension.yml:1]: Does `simple_oauth` need to be enabled too?

**Review by siliconmeadow — COMMENTED** [review]
Just the `simple_oauth` question....

File: config/default/core.extension.yml
Comment (Polynya) [reply config/default/core.extension.yml:1]: Only on Preprod, so I'll enable it manually

**Review by siliconmeadow — APPROVED** [review]
LGTM

**Pull request merged**